packages/taquito-utils/src/errors.ts

```diff
@@ -58,7 +58,7 @@ export class InvalidProtocolHashError extends ValidationError {
  *  @category Error
  *  @description General error that indicates a failure when trying to convert data from one type to another
  */
-export class ValueConversionError extends ValidationError {
+export class ValueConversionError extends ParameterValidationError {
```

Comment:
I don't think this should be a ParameterValidationError. Where is this usually thrown?

Reply:
I think this is usually thrown in utils package. So something like TaquitoUtilsError makes more sense here